app.js

```diff
@@ -3,6 +3,9 @@ const express = require('express');
 const path = require('path');
 const cookieParser = require('cookie-parser');
 const bodyParser = require('body-parser');
+const LocalStrategy = require('passport-local').Strategy;
```

Comment:
You aren't using passport-local in this file. You seem to only use it in routes/index so you only need it there.